<title>: Fix database selection when using the REST API

GuilleX7 opened the pull request:
<type>: backend
<body>: There were some problems when selecting the root database in order to search for instances

GuilleX7 committed:
<title>: Fix database selection when using the REST API
<type>: backend
<body>: There were some problems when selecting the root database in order to search for instances

CarmenMariaMP reviewed (changes requested):
La integración continua ha fallado.

GuilleX7 committed:
<title>: Fix CI integration
<type>: backend
<body>: The instance of the MySQL database was not given the right credentials

GuilleX7 committed:
<title>: Fix environment variables for CI
<type>: backend
<body>: Some environment variables were pointing to a wrong instance of the MySQL database

GuilleX7 committed:
<title>: Fix instance test
<type>: backend
<body>: The instance test was given wrong credentials to access the database

CarmenMariaMP reviewed (approved):
Integración continua funciona correctamente.
